crates/apollo-compiler/Cargo.toml

```diff
@@ -31,3 +31,4 @@ uuid = { version = "1.1", features = ["serde", "v4", "js"] }
 [dev-dependencies]
 expect-test = "1.1"
 anyhow = "1.0"
+notify = "4.0.0"
```

Comment:
just fyi there is a v5 of the notify crate but there are a ton of breaking changes (i have not updated awc or rover yet)

Reply:
yea i saw, totally didn't want to figure out how to do the watcher in a brand new and exciting way lol